src/obis-observer/commands/downlink/ReadArchive.ts

```diff
@@ -0,0 +1,107 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
+import {DOWNLINK} from '../../constants/directions.js';
+import {archiveTypes} from '../../constants/index.js';
+
+
+/**
+ * IReadArchiveParameters command parameters
+ */
+interface IReadArchiveParameters extends ICommandParameters {
+    archiveType: number,
+    time: number
+}
+
+
+const COMMAND_ID = 0x11;
+const COMMAND_SIZE = REQUEST_ID_SIZE + 5;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'request detailed archive from 2023-12-22 23:40:00 GMT',
+        parameters: {
+            requestId: 33,
+            archiveType: archiveTypes.DETAILED,
+            time: 756604800
+        },
+        hex: {header: '11', body: '21 01 2d 18 df 80'}
+    },
+    {
+        name: 'request summary archive from 2023-12-22 23:40:00 GMT',
+        parameters: {
+            requestId: 34,
+            archiveType: archiveTypes.SUMMARY,
+            time: 756619200
+        },
+        hex: {header: '11', body: '22 02 2d 19 17 c0'}
+    }
+];
+
+
+/**
+ * Downlink command.
+ *
+ * @example
+ * ```js
+ * import ReadArchive from 'jooby-codec/obis-observer/commands/downlink/ReadArchive.js';
+ *
+ * const parameters = {
+ *     requestId: 34,
+ *     archiveType: 2,
```

Comment:
would be nice to use constants here as well

Reply:
constants added